### ISSUE TITLE
Backslashes being added to contact names after syncing profile pictures

### ISSUE THREAD

**Issue by m-mclean** [skjnldsv/Owncloud-FBSync#10]
I'm encountering an issue where backslashes are being added to the names of my contacts after syncing their profile pictures. For example, "Foo Bar" will become "Foo Bar\".

I'm not seeing anything in the ownCloud logs related to this... I'm new to ownCloud, so I don't know if I should be looking at something specific. What sort of data can I provide that could be useful?

Ubuntu Server 14.04.3, MySQL 5.5, ownCloud 8.2.1, Contacts 0.5.0.0, Fb Sync 1.7.1

**Comment by skjnldsv** [comment]
Hi!
Hum, that's strange.
Could you download a single vcf card and check in which field is added a backslash?
Regards.

**Comment by m-mclean** [comment]
I think I've figured this out, and the problem is (somewhat) unrelated to Fb Sync.

Like I've said earlier, I'm new to ownCloud. I imported my contacts from Google using a Google-formatted CSV. I tried a fresh import in that format and then downloaded the VCF from ownCloud. The ownCloud VCF has "Foo Bar" listed as the FN, and "Bar\;Foo;;;;" listed as the N. After linking Foo Bar to his Facebook accound in Fb Sync, the FN field becomes "Foo Bar\". It seems that when Fb Sync adds the FBID field to the contact, the FN field somehow picks up the backslash from the N field.

I deleted all of my contacts and re-imported them using a big VCF instead of a CSV this time. I downloaded a few contacts and could not find any stray backslashes, and Fb Sync didn't cause any issues this time.

So in short, I think this problem was caused by ownCloud producing bad VCFs when importing from a CSV. I'm assuming Fb Sync only modifies the FBID field of a contact and doesn't touch the name fields, so I think this problem is outside the scope of Fb Sync.

Thank for for the response, though!

**Comment by skjnldsv** [comment]
That's what I though too.
I don't touch the fields used for the names. And because I'm using some scripts from the contacts+ app, I would have found this pretty strange.

Glad everything's fine now! :)
Thanks for your feedback!
